developing_features/guides/new_project.md

```diff
@@ -0,0 +1,44 @@
+# [Developing Features](../developing_features.md) / Guides / Setting Up a New Project
+
+
+#### Checklist
+
+###### For a Public project
+
+ - Generate a new gem with `bundle gem abc`
+ - Commit it, before changing anything, as `[skip] Initial commit from Bundler 1.16.1 (1m)` (giving the appropriate version of Bundler)
+ - [Create a new repo on GitHub](https://github.com/new) (e.g. `cph/abc`)
+ - [Enable branch protection](https://help.github.com/articles/configuring-protected-branches/) on `master` and turn on these settings:
+    - Require pull request reviews before merging
+    - Require status checks to pass before merging
+       - Require branches to be up to date before merging
+       - Select Travis CI - Pull Request
+       > You won't be able to do this until you've pushed commits and had one build run on Travis.
+
+###### For a Private project
+
+ - Generate a new Rails project with `rails new xyz`
+ - Commit it, before changing anything, as `[skip] Initial commit from Rails 5.2.2 (1m)` (giving the appropriate version of Rails)
+ - [Create a new repo on GitHub](https://github.com/new) (e.g. `cph/xyz`)
```

Comment:
TODO: add steps for configuring **Collaborators & teams**

Reply:
<img width="748" alt="screen shot 2019-01-11 at 8 39 28 am" src="https://user-images.githubusercontent.com/151907/51045593-95d55600-1589-11e9-8621-57065df32806.png">